pkg/webhook/validator_test.go

```diff
@@ -573,19 +573,7 @@ func TestValidatePodSpec(t *testing.T) {
 				},
 			},
 		),
-		want: func() *apis.FieldError {
```

Comment:
If a secret (imagePullSecrets) or service account doesn't exist, the package go-containerregistry ignores these errors https://github.com/google/go-containerregistry/issues/1471.